### ISSUE TITLE
Mirror Stream to H264 V4L2 device

### ISSUE THREAD

**Issue by br33333** [antimof/UxPlay#26]
Would like to forward the mirror stream to an application which accepts h264 streams only from V4L2 devices.

Perhaps UxPlay -> v4l2loopback via gstreamer ? 

Or is gstreamer not necessary as middleman ? 

Would appreciate any help and/or insight as to how either of these could be accomplished (setting up the V4L2loopback device and configuring GST to sending the stream to it or sending stream directly to the V4L2 device).

Thanks!

**Comment by antimof** [comment]
gstreamer v4l2sink is your friend....

**Comment by Bob2345de** [comment]
Yes, got that. Have no clue what I am doing in this space. Haven’t understood how/where to configure uxplay/gstreamer to send the mirror stream to a device or even what the default device it sends the streams to.
Any further help would be very much appreciated.

**Comment by fduncanh** [comment]
its not a current option, but you could probably easily modify the gstreamer pipeline that is created in   
UxPlay/lib/video_renderer_gstreamer.c aftre reading  the first few of the  gstreamer tutorials

(There is an updated UxPlay at pr #58 )
